mipengine/controller/algorithm_executor.py

```diff
@@ -137,6 +140,8 @@ def _instantiate_algorithm_execution_interface(self):
             algorithm_parameters=self._algorithm_execution_dto.algorithm_request_dto.parameters,
             x_variables=self._algorithm_execution_dto.algorithm_request_dto.inputdata.x,
             y_variables=self._algorithm_execution_dto.algorithm_request_dto.inputdata.y,
+            pathology=self._algorithm_execution_dto.algorithm_request_dto.inputdata.pathology,
```

Comment:
This is correct but we shouldn't change it just here. The problem stems from `AlgorithmInputDataDTO` from which the name "pathology" is propagated downstream. We need a separate PR to completely remove "pathology" from the system. This can be done once `mipdb` is deployed, which also introduces "data_model" into the DB.